### DISCUSSION TITLE
When I finish pull request, How to delete the fork repository in my github.

### DISCUSSION THREAD

**Discussion by improve100** [magit/forge#559] [Support] [answered]
When I follow this process to clone a repo, fork it to GitHub, fix an issue, submit a PR, and have it merged. I want to delete the forked repo on my GitHub. Can I do it on Magit Forge or do I have to go to the GitHub website to delete it?

**Comment by tarsius** [comment, marked as answer]
No, that is not supported. I have decided against adding support for such very destructive actions.

**Reply by improve100** [reply]
Okay,  we can only delete it through the github website.
